src/main/java/filemanager/directorytracker/ScheduleFileDirectory.java

```diff
@@ -36,48 +36,18 @@ public ScheduleFileDirectory(JsonReadService readService, InteractionGroupServic
     }
 
     public void goThroughToCheckFile() {
-        this.goThroughToCheckFile(LocalDateTime.now());
-    }
-
-    public void goThroughToCheckFile(LocalDateTime time) {
-        interactionsMap = new HashMap<>();
-        interactions = new ArrayList<>();
-        Path allPath = Paths.get(getRootFolder() + getEnvironment() + "/" +
-                time.getYear() + "/" +
-                time.getMonthValue() + "/" +
-                time.getDayOfMonth() + "/" +
-                time.getHour());
-        if (Files.exists(allPath)) {
-            try {
-                Files.walk(allPath, FileVisitOption.FOLLOW_LINKS)
-                        .filter(file -> Files.isRegularFile(file) && matchPattern(file.toString()))
-                        .forEach(path -> {
-                            try {
-                                readService.readJson(new FileInputStream(path.toString()), interactions);
-                            } catch (IOException e) {
-                                e.printStackTrace();
-                            }
-                        });
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
-            groupService.groupByClient(interactions, interactionsMap);
-            interactionsMap.forEach((clientName, interactions1) -> {
-                String temporaryPath = getOutputPath() + clientName;
-                try {
-                    writeService.writeXmlFile(interactions1, temporaryPath);
-                } catch (IOException e) {
-                    e.printStackTrace();
-                }
-
-            });
-        }
+        Command command = new Command();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH");
+        String dateStr = LocalDateTime.now().format(formatter);
+        command.setDate(dateStr);
```

Comment:
I actually find this a bit weird. Please make command to contain LocalDateTime instead of String and make it part of ```ClientResource```'s responsibility to fill command with proper date. I'd also expect it to be handled by Jackson for you, so no manual work should be needed there.

Reply:
Another thing about this class is that I don't think that you need separate method for scheduled execution. You may change your scheduling task to create command object for you and pass it along with related method call.